compiler/parser/chapel.ypp

```diff
@@ -805,43 +805,55 @@ forwarding_stmt:
 extern_export_decl_stmt:
   TEXTERN TRECORD ident_def opt_inherit TLCBR class_level_stmt_ls TRCBR
     {
+      const char* comment = context->latestComment;
+      context->latestComment = NULL;
```

Comment:
I kinda suspect this won't work.  I'm pretty sure we explicitly used the comment on `TRECORD` due to the presence of comments on symbols defined within the class itself.  There should be tests to lock this in for ordinary records/classes, which I expect you will see fail

Reply:
If that theory is correct, you ought to be able to grab the comment from `TEXTERN` instead?